MTFO/Components/HotReloader.cs

```diff
@@ -1,91 +1,269 @@
+        /// </summary>
+        private void CleanIconsOfTier()
+        {
+            CleanIconsOfTier(Rundown.m_expIconsTier1);
+            CleanIconsOfTier(Rundown.m_expIconsTier2);
+            CleanIconsOfTier(Rundown.m_expIconsTier3);
+            CleanIconsOfTier(Rundown.m_expIconsTier4);
+            CleanIconsOfTier(Rundown.m_expIconsTier5);
+        }
+
+        /// <summary>
+        /// Destroy expedition buttons of a tier
+        /// </summary>
+        private void CleanIconsOfTier(Tier tier)
+        {
+            foreach (var icon in tier)
+            {
+
+                GameObject.Destroy(icon.gameObject);
+            }
+        }
+
+        /// <summary>
+        /// Replace the rundown data
+        /// </summary>
+        private void TryPlaceRundown()
+        {
+            Rundown.m_currentRundownData = this.m_currentRundownData;
+            if (Rundown.m_currentRundownData != null)
+            {
+                Rundown.PlaceRundown(Rundown.m_currentRundownData);
+                Rundown.m_dataIsSetup = true;
+            }
+        }
+
+        private bool HasValidRundown => GameDataBlockBase<RundownDataBlock>.s_blockByID.ContainsKey(Global.RundownIdToLoad);
+        private RundownDataBlock m_currentRundownData => GameDataBlockBase<RundownDataBlock>.GetBlock(Global.RundownIdToLoad);
+        private CM_PageRundown_New Rundown;
+    }
+
+    class HotGearManager : HotManagerBase
+    {
+        public HotGearManager() : base() { }
+
+        public override void Reload(int id)
+        {
+            GearManager.Current.m_offlineSetupDone = false;
+            this.CleanGearIcons();
+            this.CleanGearSlots();
+            GearManager.Current.SetupGearContainers(); //Re-initialize most data parameters
+            this.LoadOfflineGearDatas();
+            GearManager.GenerateAllGearIcons();
+            GearManager.Current.m_offlineSetupDone = true;
+            Log.Message("Reloaded Gear");
+        }
+
+        /// <summary>
+        /// Clears all RenderTextures for every persistentID of the gear. Note: does not destroy the texture objects!
+        /// </summary>
+        private void CleanGearIcons()
+        {
+            GearManager.m_allGearWithPostedIconJobs.Clear();
+            foreach (var instance in GearManager.Current.m_allGearIconTexturesPerInstanceKey)
+                instance.Clear();
+        }
+
+        /// <summary>
+        /// Destroy all selectable gear in the lobby, leaving nothing selectable
+        /// </summary>
+        [Obsolete]
+        private void CleanGearLobbySlots()
+        {
+            if (CM_PageLoadout.Current != null)
+            {
+                var inventoryItems = CM_PageLoadout.Current.m_popupAlign.gameObject
+                                   .GetComponentsInChildren<CM_InventorySlotItem>(true);
+                foreach (var item in inventoryItems) GameObject.Destroy(item.gameObject);
+            }
+        }
```

Comment:
This is marked private and never used, even though it's marked as Obsolete. If kept for external use, make it `public`

Reply:
It completely destroys all inventory items from the lobby making nothing selectable, so it's not really useful for anything at the moment, hence the `Obsolete` since I figured out a better way to get the gear to refresh in the lobby